### ISSUE TITLE
tab completion support similar to `ssh` and `sftp`

### ISSUE THREAD

**Issue by jzeus** [libfuse/sshfs#139]

**Comment by billziss-gh** [comment]
I do not think this is applicable to sshfs or sshfs-win.

**Comment by Nikratio** [comment]
Thanks for the suggestion! Having completion support would indeed be nice.

That said, please note that sshfs is a fully volunteer driven project. This means that new features and bugfixes are implemented only when someone has a personal interest in them and therefore also does the necessary work. Feature requests are therefore in general unlikely to actually result in the feature being implemented.

Would you be interested in working on this? If not, I will close this issue in a few days (since the purpose of the issue tracker is to coordinate ongoing and planned work, rather than enumerate all the work that could possibly be done).

**Comment by jzeus** [comment]
yes I'll be looking to implement this, will be sending a pull request sometime soon

**Comment by Jakuje** [comment]
Would not it make sense to add the tab completion directly to https://github.com/scop/bash-completion rather than shipping it separately in sshfs?
